html/captive-portal/lib/captiveportal/PacketFence/Controller/Signup.pm

```diff
@@ -22,7 +22,7 @@ use POSIX;
 use URI::Escape::XS qw(uri_escape);
 use pf::iplog;
 use pf::node;
-use pf::person qw(person_modify);
+use pf::person qw(person_modify @FIELDS);
```

Comment:
There's no need to import '@FIELDS' since you use the complete name in line 34 and it is the right way to do :)